.gitattributes

```diff
@@ -20,6 +20,7 @@ box.json export-ignore
 check_trailing_spaces.sh export-ignore
 phpmd.xml export-ignore
 phpunit.xml.dist export-ignore
+.circleci export-ignore
```

Comment:
Can we move it a little bit up to keep alphabetic order? And also add `/` after the directory name to point it is directory?